File: PB - More Exercises Python/2.0 Conditional Statements - More Exercises/04. Transport Price.py
km, day_type, price = int(input()), input(), 0

if km < 20:
    if day_type == 'day':
        price = 0.79 * km + 0.70
    else:
        price = 0.90 * km + 0.70
elif 20 <= km < 100:
    price = km * 0.09
else:
    price = km * 0.06

print(f'{price:.2f}')


'''
7
night
'''

File: PB - More Exercises GO/2.0 Conditional Statements - More Exercises/04. Transport Price.go
package main

import "fmt"

func main() {
	var km float64
	var day string
	fmt.Scan(&km, &day)
	price := 0.0

	if km < 20 {
		if day == "day" {
			price = 0.79*km + 0.70
		} else {
			price = 0.90*km + 0.70
		}
	} else if 20 <= km && km < 100 {
		price = km * 0.09
	} else {
		price = km * 0.06
	}
	fmt.Printf("%.2f", price)
}

/*
name   :04. Transport Price
input  :7 night
output :7.00
*/
